src/controllers/admin/webhook.ts

```diff
@@ -280,7 +280,7 @@ export class WebhookController {
 
 		const customer = await CustomerService.instance.findbyPaymentProviderId(data.paymentProviderId);
 		if (customer) {
-			this.syncLogtoRoles(operation, customer.customerId, '');
+			await this.syncLogtoRoles(operation, customer.customerId, '');
```

Comment:
should the product name be an empty string? @jay-dee7

Reply:
This is only valid for Cancel subscription.